### PR TITLE
Maps fix

### LINKED ISSUE #468: Route Map and API key
Route map looks like this for all routes (grayed out with "For development purposes only").
screen shot 2018-09-13 at 1 45 08 pm

Google prompts with:
screen shot 2018-09-13 at 1 46 47 pm

@werebus researched it and found it has to do with our API key. Either we're using it too much, didn’t pay enough, or something else.

### PR DESCRIPTION
**Pull request by mboneil10**
Closes #468

### CODE REVIEW

**Review by dfaulken — COMMENTED** [review]
This seems correct, and is more along the lines of what I was expecting the code to look like, but I will leave explicit approval to someone with more knowledge.

**Comment by Anbranin** [comment]
> I will leave explicit approval to someone with more knowledge.

lol who

File: src/providers/map.service.ts [hunk 1]
```diff
@@ -1,21 +1,35 @@
...
 export class MapService {
   map;
 
+  gmaps_key;
   currentLocation;
   options = { timeout: 5000, enableHighAccuracy: true };
   markers = [];
   windows = [];
-  constructor() {}
+  constructor(public platform: Platform) {}
```
Comment (Anbranin) [inline src/providers/map.service.ts:15]: Did you figure out what this does?

Comment (Anbranin) [reply src/providers/map.service.ts:15]: Oh nvm I see it in the docs. Nice.

**Review by Anbranin — APPROVED** [review]

**Pull request merged**